frontend/assets/theme/base.scss

```diff
@@ -9,9 +9,13 @@ body,
 .v-sheet.v-list {
   background-color: var(--v-accent-base) !important;
 }
-.v-btn.v-btn--has-bg {
+.theme--dark .v-btn.v-btn--has-bg {
   background-color: #1E1E1E !important;
 }
+.theme--light .v-btn.v-btn--has-bg {
+    background-color: var(--v-primary-base);
+}
+
```

Comment:
I don't see how this is the right way to change button colors. This changes **all** button colors correct? Are the vuetify color props on buttons or the helper color css classes going to have any effect with this.

Reply:
It doesn't change all button colors – only some specific ones which have that `btn--has-bg` class. Since `#1E1E1E` was already hard-coded as a button background, I hard-coded var(--v-primary-base) for the light theme.

I don't see why any light theme button should have #1E1E1E as a background color, so isn't it better to change _all_ of those button colors there?